.github/workflows/deploy-beta.yml

```diff
@@ -11,11 +11,20 @@ jobs:
   deploy-tag:
     name: Deploy BETA Feature
     runs-on: ubuntu-latest
-    if: startsWith(github.ref, 'refs/heads/beta/')
+    if: startsWith(github.ref, 'refs/heads/beta/') || startsWith(github.ref, 'refs/tags/bugbash')
     steps:
       - name: Extract feature name from branch
         shell: bash
-        run: echo "branch=$(echo ${GITHUB_REF#refs/heads/beta/})" >>$GITHUB_OUTPUT
+        run: |
+          source_branch_name=${GITHUB_REF##*/}
+          RELEASE_TYPE=beta
+          grep -q "bugbash/" <<< "${GITHUB_REF}" && RELEASE_TYPE=bugbash
+          FEATURE_NAME=${source_branch_name#bugbash/}
+          FEATURE_NAME=${FEATURE_NAME#beta/}
```

Comment:
Is this a mistake? The release type is already part of the final CDN path.

Reply:
notsure I understand this comment steps.extract_branch.outputs.branch_type exposes the value of RELEASE_TYPE